.github/workflows/publish-extension-nightly.yml

```diff
@@ -79,7 +79,16 @@ jobs:
         env:
           VSCE_PAT: ${{ secrets.VSCODE_PUBLISH_PWD }}
         working-directory: ./packages/plugin-core
-        run: yarn deploy:vscode
+        run: |
+          npx verdaccio > verdaccio.log  2>&1 &
+          sleep 30
+          echo "checking verdaccio log"
+          cat verdaccio.log
+          echo "Setting registry to local"
+          yarn config set registry http://localhost:4873
+          npm set registry http://localhost:4873/
+          echo "running deploy:vscode"
+          yarn deploy:vscode
```

Comment:
so did a test, looks like the following command still tries to fetch the dependencies. 

```
vsce publish --yarn
```

confirm that its not actually installing them but it will fail if it its not able to access them

Reply:
can we add this as a comment to this pr?

Reply:
Did you mean a comment in the PR description itself, or in the workflow file? (I can do both)

Reply:
workflow file